index.js

```diff
@@ -72,7 +72,7 @@ class ServerlessAppsyncPlugin {
       'graphql-playground:run': () => this.runGraphqlPlayground(),
       'deploy-appsync:deploy': generateMigrationErrorMessage('deploy-appsync'),
       'update-appsync:update': generateMigrationErrorMessage('update-appsync'),
-      'before:deploy:deploy': () => this.addResources(),
+      'after:aws:package:finalize:mergeCustomProviderResources': () => this.addResources(),
```

Comment:
@kinyat I am not familiar with this hook but I see it is also used by `serverless-plugin-split-stacks`
Does this means that `serverless-plugin-split-stacks` has to be placed **after** `serverless-appsync-plugin` in serverless.yml file plugins list? (so that first this plugin adds teh resource, and then the other plugin splits them).
If so, this could be specified somewhere in the readme...

Reply:
Cool. I will update the readme for the usage with the split stacks plugin

Reply:
I have updated the readme.md